tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -14,21 +14,68 @@ import (
 	"github.com/openshift/odo/tests/helper"
 )
 
-const (
-	CI_OPERATOR_HUB_PROJECT = "ci-operator-hub-project"
-)
-
 var _ = Describe("odo service command tests for OperatorHub", func() {
 
+	var project string
+
 	BeforeEach(func() {
 		SetDefaultEventuallyTimeout(10 * time.Minute)
 		SetDefaultConsistentlyDuration(30 * time.Second)
-		helper.CmdShouldPass("odo", "project", "set", CI_OPERATOR_HUB_PROJECT)
 		// TODO: remove this when OperatorHub integration is fully baked into odo
 		os.Setenv("ODO_EXPERIMENTAL", "true")
 	})
 
+	preSetup := func() {
+		project = helper.CreateRandProject()
+		helper.CmdShouldPass("odo", "project", "set", project)
+
+		// wait till oc can see the all operators installed by setup script in the namespace
+		ocArgs := []string{"get", "csv"}
+		helper.WaitForCmdOut("oc", ocArgs, 1, true, func(output string) bool {
+			return strings.Contains(output, "NAME")
+		}, true)
```

Comment:
This is required because every time a user creates a new project, it takes OpenShift a few seconds before it can make the clusterwide operators available to the newly created namespace.